### PR TITLE
Online Soccer Manager | Presence

### PR DESCRIPTION
**Pull request by marinofranz**
image

### CODE REVIEW

File: websites/O/Online Soccer Manager/presence.ts [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        case "/User/Profile": {
+            const SelfuserName = document.querySelector("#user-profile-name-container > div:nth-child(2) > div");
+            presenceData.details = "Viewing profile:";
+            presenceData.state = SelfuserName.textContent;
+            break;
+        }
+
+        default:
+            presenceData.details = "Viewing:";
+            presenceData.state = "";
```
Comment (doomlerd) [inline websites/O/Online Soccer Manager/presence.ts:233]: Don't set state if it's empty.

**Comment by Xzer0o0** [comment]
hi , there is a lot of problems in it , first it doesn't work when the language is not english , for example when the link is "ar.onlinesoccermanager.com" and not "en.onlinesoccermanager.com" it stop working and refuse to show anything , second problem is when you are browsing "career" it write : browsing NOT FOUND. , and i suggest when u are browsing dashboard it write : browsing (LIVERPOOL TEAM) "for exampe" , or it just say : browsing dashboard and not (home page)
thank you.

**Comment by marinofranz** [comment]
Hi, at the moment I am not doing support for other languages, I might do that in the future. The second problem, I must have missed the career page, so sorry for that. I will be updating that now.

**Comment by marinofranz** [comment]
@Xzer0o0  One question, does the other language pages look any different?

**Comment by marinofranz** [comment]
See pull request: #1775

**Pull request merged**